package.json

```diff
@@ -8,13 +8,14 @@
     "yarn": ">= 1.3.2"
   },
   "scripts": {
-    "docs": "./node_modules/.bin/jsdoc --configure jsdoc.json",
-    "lint": "./node_modules/.bin/eslint **/*.js --ignore-pattern node_modules/",
-    "lint:watch": "./node_modules/.bin/chokidar --initial './lib' './test' './examples' -c 'yarn lint'",
-    "test": "./node_modules/.bin/mocha test --reporter spec",
-    "test:watch": "./node_modules/.bin/chokidar --initial './lib' './test' './examples' -c 'yarn test'",
+    "docs": "jsdoc --configure jsdoc.json",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "lint:watch": "chokidar --initial './lib' './test' './examples' -c 'yarn lint'",
+    "test": "mocha test --reporter spec",
+    "test:watch": "chokidar --initial './lib' './test' './examples' -c 'yarn test'",
     "watch": "yarn test:watch & yarn lint:watch",
-    "coveralls": "./node_modules/.bin/nyc yarn test && node_modules/.bin/nyc report --reporter=text-lcov | node_modules/.bin/coveralls"
+    "coveralls": "nyc yarn test && nyc report --reporter=text-lcov | coveralls"
```

Comment:
Why shouldn't this be `./node_modules/.bin/xxx`?

Reply:
`./node_modules/.bin/<whatever>` is what you would use to invoke from the command-line.

yarn and npm already have built-in handling for locally-installed executable in `package.json` -- this feature goes back as far as I've been using npm for 5+ years. 

So: `./node_modules/.bin/...` is redundant.